CONTRIBUTING.md

```diff
@@ -147,7 +147,9 @@ This list gives a brief summary of what each function does. Examples can be foun
 
 Function | Note
 -------- | -----
-`regexp` | Allow regex find and/or replace on a given field. Useful to extract house number/street/city/region etc when the source has them in a single field
+`prefix_number` | Allow number to be extracted from a single field using an internal convenience regex.
```

Comment:
A minor nit: do you think it would make sense to remove reference to it being a regular expression behind the scenes? This way we can swap out the backend implementation with something non-regex-ey later. Or maybe it doesn't matter...